downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/Biopax.java

```diff
@@ -0,0 +1,211 @@
+	  	// Temporary location where BioPAX files are stored
+	  	String biopaxDir = releaseNumber + "_biopax";
+        File biopaxDirFile = new File(biopaxDir);
+        if (!biopaxDirFile.exists()) {
+        	biopaxDirFile.mkdir();
+        }
+        
+        JSONParser parser = new JSONParser();
+        JSONObject speciesFile = (JSONObject) parser.parse(new FileReader(pathToSpeciesConfig));
+        @SuppressWarnings("unchecked")
+		Set<String> speciesKeys = speciesFile.keySet();
+        // First, generate owl files. This particular step requires a local maven installation of the PathwayExchange jar (see README.md for DownloadDirectory)
+        // We want to run BioPAX level 2 and BioPAX level 3, so the below loop starts with '2'
+		for (int biopaxLevel = 2; biopaxLevel < 4; biopaxLevel++) {
+			for (String speciesKey : speciesKeys) {
+				String speciesName = (String) ((JSONArray) ((JSONObject) speciesFile.get(speciesKey)).get("name")).get(0);
+				if (biopaxLevel == 2 && runBioPAX2) {
+					logger.info("Generating BioPAX2 " + speciesName + " owl file...");
+					generateBioPAXFile(host, database, username, password, port, biopaxDir, speciesName, biopaxLevel);
+				} else if (biopaxLevel == 3 && runBioPAX3) {
+					logger.info("Generating BioPAX3 " + speciesName + " owl file...");
+					generateBioPAXFile(host, database, username, password, port, biopaxDir, speciesName, biopaxLevel);
+				}
+			}
+			String outfile = "";
+			if (biopaxLevel == 2) {
+				outfile = "biopax2.zip";
+			} else if (biopaxLevel == 3) {
+				outfile = "biopax.zip";
+			}
+			validateBioPAX(biopaxDir, biopaxLevel, releaseNumber, outfile);
+			// Move zip file and remove temp directory
+			String outpathBiopax = releaseNumber + "/" + outfile;
+			Files.move(Paths.get(outfile), Paths.get(outpathBiopax), StandardCopyOption.REPLACE_EXISTING);
+			FileUtils.deleteDirectory(new File(biopaxDir));
+		}	
+  	}
+  	
+    //Generate BioPAX files using the appropriate SpeciesAllPathwaysConverter function found in Pathway-Exchange
+	private static void generateBioPAXFile(String host, String database, String username, String password, String port, String biopaxDir, String speciesName, int biopaxLevel) throws Exception {
+		
+		if (biopaxLevel == 2) {
+			SpeciesAllPathwaysConverter converter = new SpeciesAllPathwaysConverter();
+			converter.doDump(new String[]{host, database, username, password, port, biopaxDir, speciesName});			
+		} else if (biopaxLevel == 3) {
+			SpeciesAllPathwaysLevel3Converter level3converter = new SpeciesAllPathwaysLevel3Converter();
+			level3converter.doDump(new String[]{host, database, username, password, port, biopaxDir, speciesName});
+		}
+	}
+	
+	// Next, once all species files for the current BioPAX level have been generated, we must validate them to make sure they are appropriately formatted
+	private static void validateBioPAX(String biopaxDir, int biopaxLevel, String releaseNumber, String outfile) throws Exception {
```

Comment:
It doesn't look like releaseNumber and outfile are used here, can they be removed?

Reply:
Yes they can, artifacts from before the refactor I think